### PR TITLE
Rename output files

### PR DESCRIPTION
**Pull request by dagewa**
* I have not done a --full-regression test yet
* I am unsure if Driver/test_timing.py also requires changes
* Handlers/Phil.py has input.json=. Perhaps this should be input.expt=?

### CODE REVIEW

File: command_line/to_shelx.py [hunk 1]
```diff
@@ -192,7 +192,7 @@ def to_shelx(hklin, prefix, compound="", options=None):
     unit_cell_esds = None
     cell_data = None
     if options and options.cell:
-        if options.cell.endswith(".json"):
+        if options.cell[-5:] in (".expt", ".json"):
```
Comment (Anthchirp) [inline command_line/to_shelx.py]:
```suggestion
        if options.cell.endswith((".expt", ".json"))
```

File: command_line/overload.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def build_hist(nproc=1):
     if len(sys.argv) >= 2 and sys.argv[1].startswith("nproc="):
         nproc = int(sys.argv[1][6:])
         sys.argv = sys.argv[1:]
-    if len(sys.argv) == 2 and sys.argv[1].endswith(".json"):
+    if len(sys.argv) == 2 and sys.argv[1][-5:] in (".expt", ".json"):
```
Comment (Anthchirp) [inline command_line/overload.py]:
```suggestion
    if len(sys.argv) == 2 and sys.argv[1].endswith((".expt", ".json")):
```

File: command_line/to_shelx.py (hunk 1 above)
Comment (dagewa) [reply command_line/to_shelx.py]: Nice, I did not know `endswith` can take multiple arguments

**Comment by Anthchirp** [comment]
`Driver/test_timing.py` does not need to be changed. Command line values in there are dummy values that come out of a real run.

**Pull request merged**